privacyidea/api/lib/postpolicy.py

```diff
@@ -170,14 +170,14 @@ def after_request(response):
     else:
         response_object = response
     try:
-        content = json.loads(response_object.data)
```

Comment:
In other versions than 2.5 the json object already exists in the ``response_object``.
What would be the speed difference, when we do an additional jsonify?
Could it make sense to only do a ``loads`` if the json object does not exist, yet?